rootfs/etc/owncloud.d/95-univention.sh

```diff
@@ -130,4 +130,4 @@ sed -i "s#);#  'log_rotate_size' => 104857600,\n&#" $OWNCLOUD_CONF/config.php
 echo "[02.DOCKER_SETUP] configuring owncloud for onlyoffice use"
 sed -i "s#);#  'onlyoffice' => array ('verify_peer_off' => TRUE),\n&#" $OWNCLOUD_CONF/config.php
 
-exit 0
\ No newline at end of file
+exit 0
```

Comment:
This script should just end with `true`, for reference: https://github.com/owncloud-docker/base/blob/master/rootfs/etc/owncloud.d/10-folders.sh#L18

Since all the files are only [sourced](https://github.com/owncloud-docker/base/blob/master/rootfs/usr/local/bin/owncloud#L7) that should be the correct behavior.

Reply:
The second script should also just end with `true` :)